packages/slate-react/src/plugins/after.js

```diff
@@ -323,7 +323,7 @@ function AfterPlugin() {
       leaves.find(r => {
         start = end
         end += r.text.length
-        if (end >= point.offset) return true
+        if (end > point.offset) return true
```

Comment:
Why this change?

Reply:
The same reason of the previous explanation:
Consider a leaf can be extended by a spell check:
```
<b>hel</b> yourself
```
will be
```
<b>help</b> yourself
```
after spell check. Because anchorOffset is measured after the spell check. It will tell that anchorOffset is 4. However, during onInput, the text is not yet changed. offset =4 will tell the slate the spell corrected leaf is yourself, not hel